### ISSUE TITLE
Update tinyMCE library

### ISSUE THREAD

**Issue by kimisgold** [omeka/Omeka#1033]
I was trying to configure a tinyMCE editor to improve its aria attributes, and it seems that ability is only available in newer versions of tinyMCE. Omeka is using tinyMCE 4, and it's now on 7. This is a small usage, but maybe it's a good time to update the library anyway?

**Comment by csidirop** [comment]
I did some tests and it should be possible to update it, see here:
![grafik](https://github.com/user-attachments/assets/c3c14b69-afbf-4071-ac90-b0ddffa9872d)
(The upgrade button can be hidden)

But it works only partially. Not exactly sure why but It only works where the old one was already loaded once. Eg. when using the Simple Pages plugin editing a page shows the updated editor. Creating a new page does not load the new editor and the console is throwing some errors. 

Same for editing metadata fields with HTML.

Just in case someone wants to test it, here what I hade done:
```
# Install newest version somewhere (npm needed)
cd /opt/ && npm install tinymce@^7

# backup the old editor and copy the new one
mv /app/application/views/scripts/javascripts/vendor/tinymce/ /app/application/views/scripts/javascripts/vendor/tinymce-backup
cp -r /opt/node_modules/tinymce/ /app/application/views/scripts/javascripts/vendor/tinymce
chown -R www-data:www-data /app/application/views/scripts/javascripts/vendor/

# hide the upgrade button:
echo '.tox-promotion{display: none;}' >> /app/application/views/scripts/javascripts/vendor/tinymce/skins/ui/oxide/skin.min.css
```

**Comment by kimisgold** [comment]
Thanks for looking at this. We're actually working on a tinymce upgrade branch that should hopefully get merged in an upcoming release.

**Comment by zerocrates** [comment]
At the moment we'll move incrementally to 5 and we'll see about further changes in the future.

**Comment by csidirop** [comment]
Thats great! But whats the reason not to go directly to the newest versions?

**Comment by zerocrates** [comment]
Just being conservative. In particular one of the settings we use is not supported in the newer versions so figuring out the consequences of that and whether we're fine with moving or need a replacement is something that requires a little more investigation.

**Comment by csidirop** [comment]
Okay, thanks for answer.
